content/100-getting-started/02-setup-prisma/100-start-from-scratch/120-mongodb/100-connect-your-database.mdx

```diff
@@ -40,12 +40,6 @@ Here's a short explanation of each component:
 - `PORT`: The port where your database server is running (typically `27017` for MongoDB)
 - `DATABASE`: The name of the database
 
-As an example, for a MongoDB database hosted on [MongoDB Atlas](https://www.mongodb.com/cloud/atlas), the [connection URL](/reference/database-reference/connection-urls) might look similar to this:
-
-```bash file=.env
-DATABASE_URL="mongodb+srv://test:test@cluster0.ns1yp.mongodb.net/myFirstDatabase"
-```
```

Comment:
Why remove this Atlas specific example?

Reply:
Actually I think that was a confusion in the issue. The link to Atlas should definitely stay

Reply:
@janpio before I merge this one, wanted to check if William answered your question or if you had any other feedback.

Reply:
Yes. I think in other guides we use a more generic URL for the first one, so I assumed that would be the case here as well.